src/main/java/org/primeframework/mvc/util/CookieTools.java

```diff
@@ -29,7 +29,8 @@ public final class CookieTools {
   public static final int HIGHEST_BIT_MASK = 0x03;
 
   /**
-   * Processes a cookie value and calls a Function to convert it to a meaningful value for the application (or Prime).
+   * Processes a cookie value (WITHOUT REQUIRING ENCRYPTED COOKIES) and calls a Function to convert it to a meaningful value
```

Comment:
I don't know if we need to be this loud about the difference. I would say we don't need javadoc on this method at all given that it is deprecated and shouldn't be used.

Reply:
Same for the other deprecated methods in here.

Reply:
Agreed. Removed the loud but kept javadoc.